### ISSUE TITLE
In-Game open links not working

### ISSUE THREAD

**Issue by MatisPrograms** [BrenoHenrike/Skua#51]
This issue is sometimes a blessing because it makes it not open in browser windows, links that can be advertisement from Artix but on the other hand, very useful features like opening the wiki page from a quest or opening the Artix news/what's new doesn't work on Skua.

![image](https://github.com/BrenoHenrike/Skua/assets/61346475/e681f98c-3c1f-4a58-a834-a536739f9339)

Could it be possible to re-enable this feature by at least opening in the user's default browser if adding web views are too hard to achieve (like the official client does) or both with a user preference setting. At the same time having a user setting to chose what types of links that can be opened via Skua could be a nice feature and not have all links opening tabs (especially during events with NPC buttons randomly opening links)

**Comment by pitchedwhite** [comment]
Agreed especially If you want to see where items and quests come from.

**Comment by SharpTheNightmare** [comment]
Will be looking into this soon yes it has been a year

**Comment by SharpTheNightmare** [comment]
I messed up the original pull so I deleted the first PR but it will be in skua 1.3
